xml/ha_hawk2_rsc_i.xml

```diff
@@ -0,0 +1,755 @@
+      default values. </para>
+     <para>To see the attributes belonging to each proposed value, hover the mouse pointer over the
+      respective value.</para>
+     <informalfigure>
+      <mediaobject>
+       <imageobject role="fo">
+        <imagedata fileref="hawk2-monitor-op.png" width="70%" format="PNG"/>
+       </imageobject>
+       <imageobject role="html">
+        <imagedata fileref="hawk2-monitor-op.png" width="60%" format="PNG"/>
+       </imageobject>
+      </mediaobject>
+     </informalfigure>
+    </step>
+    <step>
+     <para>To change the suggested <literal>timeout</literal> values for the
+       <literal>start</literal> or <literal>stop</literal> operation: </para>
+     <substeps>
+      <step>
+       <para> Click the pen icon next to the operation.</para>
+      </step>
+      <step>
+       <para>In the dialog that opens, enter a different value for the <literal>timeout</literal>
+        parameter, for example <literal>10</literal>, and confirm your change.</para>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>To change the suggested <guimenu>interval</guimenu> value for the
+       <literal>monitor</literal> operation:</para>
+     <substeps>
+      <step>
+       <para> Click the pen icon next to the operation.</para>
+      </step>
+      <step>
+       <para>In the dialog that opens, enter a different value for the monitoring
+         <literal>interval</literal>. </para>
+      </step>
+      <step>
+       <para>To configure resource monitoring also for the case that the resource is stopped:</para>
+       <substeps>
+        <step>
+         <para>Select the <literal>role</literal> entry from the empty drop-down box below, and
+          click the <guimenu>Plus</guimenu> icon next to it to add the new parameter.</para>
+        </step>
+        <step>
+         <para>From the <literal>role</literal> drop-down box, select
+          <literal>Stopped</literal>.</para>
+        </step>
+        <step>
+         <para>Click <guimenu>Apply</guimenu> to confirm your changes and to close the dialog for
+          the operation.</para>
+        </step>
+       </substeps>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>Confirm your changes in the resource configuration screen. A message at the top of the
+      screen shows if the action has been successful.</para>
+    </step>
+   </procedure>
+
+   <para> For the processes which take place if the resource monitor detects a failure, refer to
+     <xref linkend="sec.ha.config.basics.monitoring"/>. </para>
+   <para>
+    <remark>taroth 2015-12-04: krig - is the following still true?</remark> To view resource
+    failures, switch to the <guimenu>Status</guimenu> screen in &hawk; and select the resource
+    you are interested in. Click the magnifier icon in the <guimenu>Operations</guimenu> column to
+    view the resources <guimenu>Details</guimenu>. </para>
```

Comment:
Yes, but maybe add that there is also a "Recent Events" dialog now which shows a list of recent actions performed for a given resource. Failures are marked as red in that list.